google-cloud-tasks/samples/create_http_task.rb

```diff
@@ -0,0 +1,88 @@
+# Copyright 2018 Google, Inc
+# Licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
Update the copyright notice to match the following. Make sure there's a "blank" comment line after it. Same for the other files in this PR.

```ruby
# Copyright 2020 Google LLC
#
# Licensed under ...
```

Reply:
@thagomizer Can you update the copyright notice for this and all other files? See example above.